R/viz.R

```diff
@@ -383,17 +399,30 @@ gene_mutation_tally = function(maf_df,these_samples_metadata,these_genes,groupin
 #' @param colour_index Optional named character vector with a name for each gene in these_genes and a colour as the value
 #'
 #' @return data frame with counts for each gene
+#' 
+#' @import wordcloud RColorBrewer dplyr utils
```

Comment:
Same here for utils, wonder if it should be explicitly mentioned or  we can avoid it?